### DISCUSSION TITLE
Tdarr issue with dependancy handbrake

### DISCUSSION THREAD

**Discussion by mervincm** [tteck/Proxmox#1780] [Help Needed]
Installed with defaults. Modified to allow NFS mount and added NFS common. Fail in healthcheck as it tries to initiate handbrakecli.
Does the script not include handbrake?

**Comment by tteck** [comment]
The script's functionality is straightforward and easy to understand.
https://github.com/tteck/Proxmox/blob/main/install/tdarr-install.sh

**Comment by Deekerman** [comment]
I also had issues. It didnt like the handbrake version that came with it, so i installed a new one. it also didnt like the ffmpeg that came with it so i put in https://repo.jellyfin.org/releases/server/debian/versions/jellyfin-ffmpeg/6.0-7/ 

Also had to install CCextractor. with those tweaks and making sure to edit the config to point to the new programs, its running like a top. 

I also had to set up nvidia passthrough so that might have been what some issues were related to. anyways, those things added and its chuggin along.

**Reply by mervincm** [reply]
I was trying to use Intel GPU, I wonder if that is possible?

**Reply by Deekerman** [reply]
I would imagine so. my buddy uses an intel gpu but i dont know if he uses it for tdarr. just emby right now.
